_minutes/2021-10-28-wecg.md

```diff
@@ -0,0 +1,120 @@
+ * [oliver] question about stability of MV3 manifest and keys being still in flux.
+ * [giorgio] if we get these permissions, can we get blocking WR back :)
+ * [simeon] got sidetracked with permissions because of Chrome's MV3 motivations, if we were doing thing from scratch, how we would design.  Blocking WR not compatible with long-term vision of MV3 and beyond.  No proposal for missing use cases ATM.
+
+[Issue #103](https://github.com/w3c/webextensions/issues/103): Script injection in WorkerScope(s)
+
+ * [giorgio] Use case for any anti-fingerprinting extension. Inject a script before the service worker script code runs.
+ * [timothy] Makes a lot of sense, see the usecases.
+ * [tomislav] Somewhat addressable in Firefox via blocking WR, but not trivial/elegant
+ * [giorgio] yes, lots of corner cases around caching and similar.
+ * [krzystof] extension authors can move much faster than browser vendors, talking with the good guys here (community).
+ * [timothy] valid use cases, the permission model goes a long way towards
+ * [tomislav] We are considering negative permissions – allow the user to grant access to almost everything, except some specific sites. Some complexities need to be worked through (e.g. blocking top level domain doesn't necessarily block subdomains).
+ * [timothy] Might not be clear, but this is possible in Safari. You can open preferences for a site to see what extensions have access & select deny
+ * [tomislav] might need to expose an API to answer "can I access this URL?"
+ * [timothy] Supportive of this plan
+ * [nir] instead of limiting the API, bring more transparency to the user, via a dashboard of pass activity.
+ * [tomislav, kzrysztov, simeon] too spammy, not sure if it's usable for regular users.
+ * [mukul] extensions activity log in Chromium (and Edge)
+
+[Issue #103:](https://github.com/w3c/webextensions/issues/112) Add an API to trigger extension update
+
+ * [nir] sometimes critical updates are delayed up to a full day.  If extension is communitinga with a backend, it might have metadata.
+ * [simeon] we have runtime.requestUpdateCheck, tells Chrome to ping CWS.  After that, runtime.reload() would complete the update process.
+
+[carlos] Question about more frequent meetings or engaging more often offline/via issues.
+
+ * [timothy, tomislav, simeon] probably not more meetings, but we're aiming to be more responsive, feel free to ping us.
+ * [simeon] might be less available in the coming months, trying to make the most effective use of the time he does have.
+ * [alexei] Will Google send alternate or additional representatives?
+ * [simeon] Not that I'm aware.
+ * [alexei] What does this say about google's dedication to the group?
+ * [timothy, tomislav] appreciate any time you might have.
+ * [simeon] what we're doing is very important, just trying to balance with other requirements.
+
+The next meeting will be on [Thursday, November 11th, 8 AM PDT (3 PM UTC)](https://everytimezone.com/?t=618c5d00,384).
```

Comment:
```suggestion
The next meeting will be on [Thursday, November 11th, 8 AM PDT (4 PM UTC)](https://everytimezone.com/?t=618c5d00,3c0).
```

Although the time of reference (UTC) changes from 3 PM to 4 PM UTC, the meeting time doesn't change in practice.

The summer time observation changed in between the meetings. Last weekend CEST became CET, and next weekend (before the next WECG meeting) PDT changes to PST.

(I've already made this change in the source document at https://docs.google.com/document/d/1QkwhEMtMS67JBUkl_WVPZ4lRSKoWcQNlLJSf_GwSXg8/edit#heading=h.7m7i1y6b2ibm)